src/sections/Article/index.tsx

```diff
@@ -61,7 +63,10 @@ const Article = ({
 }: Props): JSX.Element => (
   <article className={styles.article}>
     {childrenPosition === 'before' && children && <div>{children}</div>}
-    <h1 className={styles.headline}>{title}</h1>
+    <div className='header'>
```

Comment:
I believe we don't have a class called `header` in global css. In case if you want to add `header` class in _Article/index.module.scss_ file in future the className should be 

```js
<div className={styles.header}>
```

since it is a _module.scss_

Reply:
I'll look into that

Reply:
I'm sorry, but I didn't find how to make this method work with the css

Reply:
May I know what you want to achieve? 

Isn't it rendering the svg in the background?

Reply:
I want to render the header like #2932

Reply:
Irrespective of div className, I believe your changes should work since you have given the `svg` as bg in `.article` class

Reply:
IDK but its working...

Reply:
if it's working , then great 🚀 . I thought there is an issue.